src/providers.erl

```diff
@@ -62,7 +62,7 @@
 new(ModuleName, State) when is_atom(ModuleName) ->
     case code:which(ModuleName) of
         non_existing ->
-            {error, io_lib:format("Module ~p does not exist.", [ModuleName])};
```

Comment:
`~p` with an atom will show up surrounded by `'` in the case of a non-ASCII char.

Reply:
It is nice to show it with the quotes if it needs those when being used, this is why it wasn't simply using `~s`. But this change is probably still fine since this isn't generating code, it is just printing the name.

Reply:
Let me know if you prefer to revert this. Otherwise we can Resolve conversation (and it'll signal to me you're Ok with the code as-is).